plugins/org.jkiss.dbeaver.tasks.ui.view/src/org/jkiss/dbeaver/tasks/ui/view/DatabaseTasksTree.java

```diff
@@ -335,9 +336,13 @@ void regroupTasks(ExpansionOptions options) {
             } else {
                 // Add task folders as parent elements, task from these folders will be added in children list
                 if (!CommonUtils.isEmpty(allTasksFolders)) {
+                    allTasksFolders.sort((o1, o2) -> o1.getName().compareToIgnoreCase(o2.getName()));
                     rootObjects.addAll(allTasksFolders);
                 }
-                List<DBTTask> allTasksWithoutFolders = allTasks.stream().filter(task -> task.getTaskFolder() == null).collect(Collectors.toList());
+                List<DBTTask> allTasksWithoutFolders = allTasks.stream()
+                    .filter(task -> task.getTaskFolder() == null)
+                    .sorted((o1, o2) -> o1.getName().compareToIgnoreCase(o2.getName()))
+                    .collect(Collectors.toList());
```

Comment:
Is sorting here essential? We can define a custom comparator for the viewer instead.